database/postgres/seed.js

```diff
@@ -0,0 +1,73 @@
+const faker = require('faker');
+const fs = require('fs');
+var i = 1;
+// const projects = [];
+//project row
+const generateOneProjectCSVRow = (id) => {
+  return `${id},` + faker.date.past().toISOString()
+}
+
+const generateOneUpdateCSVRow = (id, foreignLimit) => {
+  return `${id},"${faker.random.words()}",${JSON.stringify(faker.lorem.paragraphs())},${faker.date.past().toISOString()},${faker.random.number()},${Math.ceil(Math.random()*foreignLimit)}`
```

Comment:
What does 'foreignLimit' do here?

Reply:
foreign keys are randomized when creating the data set, the limit prevents creating a ref to a value that doesn't exist